e2e_testing/torchscript/basic.py

```diff
@@ -529,3 +529,23 @@ def forward(self, tensor):
 @register_test_case(module_factory=lambda: LogSoftmaxIntModule())
 def LogSoftmaxIntModule_basic(module, tu: TestUtils):
     module.forward(torch.randn(3, 2, 4).double())
+
+class NumToTensorModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+    ])
+
+    def forward(self):
+        return _to_tensor(1)
+
+@torch.jit.script
+def prim_NumToTensor(i: int):
```

Comment:
What's the reason for having this extra function?

Reply:
I wasn't able to properly guard the `forward` function, so this is a hackey way to get it to run.